Wrappers/Python/cil/optimisation/algorithms/PDHG.py

```diff
@@ -110,20 +110,14 @@ def set_up(self, f, g, operator, tau=None, sigma=1., initial=None):
             self.tau = 1 / (self.sigma * normK ** 2)
         
         if initial is None:
-            self.x_old = self.operator.domain_geometry().allocate()
+            self.x_old = self.operator.domain_geometry().allocate(0)
         else:
             self.x_old = initial.copy()
 
-        self.x = operator.domain_geometry().allocate(0)        
-        self.y = self.operator.range_geometry().allocate(0)
-        self.y_old = self.operator.range_geometry().allocate(0)
-        
-        self.x_tmp = self.x_old.copy()
-        
-        self.y_tmp = self.y_old.copy()
-        
-        self.xbar = self.x_old.copy()
-        
+        self.x = self.x_old.copy()
```

Comment:
is there any difference with `self.operator.domain.allocate(0)`?

Also I'd use `self.operator.domain` rather than `self.operator.domain_geometry()`, but it may have some consequences for SIRF?

Reply:
SIRF doesn't have `domain` and `range properties, so this change will have to come in the future,